topics/admin/tutorials/interactive-tools/tutorial.md

```diff
@@ -521,7 +521,7 @@ A few Interactive Tool wrappers are provided with Galaxy, but they are [commente
 >
 >    {% raw %}
 >    ```yaml
->    galaxy_tool_config_files:
+>    galaxy_tool_config_templates:
```

Comment:
```suggestion
>    galaxy_tool_config_files:
```

This is a [role variable](https://github.com/galaxyproject/ansible-galaxy/blob/9b95b2b8ce34981649ceb475550683e95b181323/defaults/main.yml#L145)

Reply:
too much regexing. thanks